### PR TITLE
Upgrade Credentials service to Django 1.11

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
LEARNER-949
Upgrade credentials service to support django 1.11

### CODE REVIEW

File: credentials/apps/credentials/tests/test_views.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ class RenderCredentialViewTests(SiteMixin, TestCase):
     PROGRAM_TYPE = 'Professional Certificate'
 
     def setUp(self):
-        super().setUp()
+        super(RenderCredentialViewTests, self).setUp()
```
Comment (clintonb) [inline credentials/apps/credentials/tests/test_views.py]: Why is this necessary?

File: credentials/settings/test.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 ]
 
 LOGGING = get_logger_config(debug=False, dev_env=True, local_loglevel='DEBUG')
+ALLOWED_HOSTS = ['testserver.fake']
```
Comment (clintonb) [inline credentials/settings/test.py]: Why is this necessary?

Comment (ahsan-ul-haq) [inline credentials/settings/test.py]: @clintonb according to documentation [here](https://docs.djangoproject.com/en/1.11/ref/settings/#allowed-hosts) unlike previous versions `ALLOWED_HOSTS` is now checked when running tests so if we don't provide this all the view test gives 400 response.

File: credentials/apps/credentials/tests/test_views.py (hunk 1 above)
Comment (ahsan-ul-haq) [inline credentials/apps/credentials/tests/test_views.py]: While running tests I came across error for this which was resolved by this update.

Comment (schenedx) [inline credentials/apps/credentials/tests/test_views.py]: @ahsan-ul-haq What error is it? Can you paste the error message here? I wonder if this is the best solution for the error?

Comment (ahsan-ul-haq) [inline credentials/apps/credentials/tests/test_views.py]:
```
TypeError: super() takes at least 1 argument (0 given)
```

Comment (clintonb) [inline credentials/apps/credentials/tests/test_views.py]: Would you confirm you are using Python 3.5? No arguments should be necessary for Python 3.5.

File: credentials/settings/test.py (hunk 2 above)
Comment (clintonb) [inline credentials/settings/test.py]: Use '*' to avoid issues in the future.

File: requirements/base.txt [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+django==1.11.1
+django-extensions==1.7.9
+django-filter==1.0.4
 django-rest-swagger[reST]==0.3.10
 django-storages==1.5.2
 django-waffle==0.11.1
 django-webpack-loader==0.4.1
-djangorestframework==3.4.7
-djangorestframework-jwt==1.9.0
+djangorestframework==3.6.3
```
Comment (clintonb) [inline requirements/base.txt]: This is a jump in two versions of DRF. Is this necessary? If so, you will need to fix the Swagger docs at `/api-docs`. I would much rather see this change in a separate PR.

**Comment by ahsan-ul-haq** [comment]
@schenedx @clintonb please review

**Comment by clintonb** [comment]
Squash the commits.

**Comment by ahsan-ul-haq** [comment]
@clintonb Done

**Comment by schenedx** [comment]
Please verify you changes on staging as soon as possible

Simon Chen
Software Engineer
________________________________

**Comment by ahsan-ul-haq** [comment]
Verified on stage.
